web_src/css/modules/animations.css

```diff
@@ -102,6 +107,62 @@ code.language-math.is-loading::after {
   }
 }
 
+
+@keyframes pulse-opacity {
+  0% {
+    opacity: 1;
+  }
+  50% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .pulse {
+    animation: pulse 2s linear;
+  }
+}
+
 .pulse {
-  animation: pulse 2s linear;
+  animation: pulse-opacity 2s linear;
+}
+
+/***
+ * Hacky half-fix for reduced motion. To properly fix this, Fomantic-UI should
+ * implement reduced motion. This was reported upstream at
+ * https://github.com/fomantic/Fomantic-UI/issues/2872. Fixing this upstream is
+ * likely going to take a while, because the codebase is not easy to work
+ * with. As reduced motion is an accessibility feature, we need to support it
+ * even if upstream doesn't.
+ */
+
+@media (prefers-reduced-motion: reduce) {
+  .transition.slide {
+    animation-duration: 0ms !important;
+  }
+
+  .transition.flip,
+  .transition.fly,
+  .transition.slide,
+  .transition.swing,
+  .transition.zoom {
+    animation-duration: 0ms !important;
+  }
+
+  .transition.shake,
+  .transition.bounce,
+  .transition.tada,
+  .transition.pulse,
+  .transition.jiggle {
+    /* Snippet taken from the glow transition */
+    -webkit-animation-duration: 2000ms;
+    animation-duration: 2000ms;
+    -webkit-animation-timing-function: cubic-bezier(0.19, 1, 0.22, 1);
+    animation-timing-function: cubic-bezier(0.19, 1, 0.22, 1);
+    -webkit-animation-name: glow;
+    animation-name: glow;
+  }
```

Comment:
Why enable these transitions and why do it in the `reduce` block? Do they not work when completely disabled?

Reply:
It's not really a transition… it's called a "static transition", and is basically an animated loop to highlight used to highlight some element. I decided to replace all the static transitions that had motion with that one, that doesn't have motion, but still highlight that something needs your attention. I haven't found a reliable way to figure out which animation/transition is actually used in the codebase though, and if those aren't used anyways, maybe I should just remove it…

Reply:
I imagine we only use a subset of Fomantic's transitions, menu slide is really the only one that comes to mind.

Reply:
i'll just not do the hack for the static transition then, to avoid confusion